src/api/searchDevice/src/v1/steps.py

```diff
@@ -1,15 +1,80 @@
+from typing import List
+
 from aws_lambda_powertools.utilities.data_classes import APIGatewayProxyEvent
+from domain.core.device import Device, DeviceType
+from domain.fhir_translation.device import create_fhir_searchset_bundle
+from domain.repository.device_repository import DeviceRepository
+from domain.repository.marshall import unmarshall_value
 from event.step_chain import StepChain
 
-from ..data.response import devices, endpoints
-
 
-def get_results(data, cache):
+# Think we require a validation decorator here.
```

Comment:
Something we should deal with now or are we happy to continue without this?

Reply:
Yes, should do that